src/main/java/org/redline_rpm/payload/Contents.java

```diff
@@ -556,6 +556,7 @@ public int getTotalSize() {
 				if ( object instanceof File) array[ x] = ( int) (( File) object).length();
 				else if ( object instanceof URL) array[ x] = (( URL) object).openConnection().getContentLength();
 				else if ( header.getType() == DIR) array[ x] = 4096;
+				else if ( object instanceof String) array[ x] = (( String) object).length();
```

Comment:
probably this would be cleaner;

`else if ( header.getType() == SYMLINK) array[ x] = (( String) object).length();`